docker-compose-dev.yml

```diff
@@ -25,7 +25,7 @@ services:
       context: .
       dockerfile: Dockerfile-dev
     ports:
-      - "5000:5000"
+      - "5001:5000"
```

Comment:
Your AI has successfully changed the port inside the container, but has failed to update the port that listens on the client. In essence, this pull request adds no new functionality and contains the same issue that you propose to fix.